Cargo.toml

```diff
@@ -24,6 +24,11 @@ aws-smithy-http = "0.36.0"
 hyper = { version = "0.14.16", features = ["stream"] }
 pin-project = "1.0.10"
 aws-config = "0.6.0"
+log = "0.4"
```

Comment:
I think we'll end up using tracing

Reply:
I don't quite understand the difference, can you create a new issue to include more details?

Reply:
BTW, we didn't print any log/trace in opendal so far, let me just remove the dep for now.